tox.ini

```diff
@@ -19,6 +19,31 @@ deps = -r{toxinidir}/requirements.txt
        -e{toxinidir}/st2reactor
 
 # Python 3 tasks
+[testenv:py36-black]
```

Comment:
NOTE: I needed to add a special tox target because ``black`` only works under Python 3.